### DISCUSSION TITLE
Engstellen

### DISCUSSION THREAD

**Discussion by funcarfreak** [Courseplay/Courseplay_FS22#2857] [Ideas]
Hi, ist es irgendwie möglich das so zu programmieren das die Helfer in Engstellen micht inner hängen bleiben...!?

MfG Daniel
![fsScreen_2023_10_15_13_24_17](https://github.com/Courseplay/Courseplay_FS22/assets/80636987/c6e5509b-d52c-4d0f-b39c-77ce6c4e1c80)

**Comment by BalticHawk** [comment]
CP has nothing to do with driving somewhere, so they can't do anything about this.
You could use AD instant of the normal FS Helper to crate a street network that works better.

**Comment by Tensuko** [comment]
Das ist das Problem von Giants, nicht unser, da können wir auch nichts dran ändern.

**Reply by funcarfreak** [reply]
ah, ok... so tief stecke ich da nicht drin...
Danke
